<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>240</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minLcdNumber">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secLcdNumber">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="quitPushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resumePushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="scoreLcdNumber">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
